app/controllers/account/charts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Account::ChartsController < ApplicationController
+  before_action :authorize_admin!
+  def chart_for_user_registration
+    chart_data = current_tenant.users
+    render json: chart_data.group_by_week(:created_at, range: time_range).count
+  end
+
+  def chart_for_companies
```

Comment:
still inconsistent naming
try to keep naming in reatlike readable style..
as is now you have
```
GET /account/charts/charts_for_companies
                          ^^^^^^^^^^^^^^^^^^^^
                                  sounds?
mb something like this?
GET /account/charts/companies
```

Reply:
I would recomment to "chart_created_companites" or without "chart_" to be more clear what it returns